File: eco-extensions/attackspeed/src/main/java/com/willfp/ecoskills/attackspeed/AttackSpeedConfig.java
package com.willfp.ecoskills.attackspeed;

import com.willfp.eco.core.PluginLike;
import com.willfp.eco.core.config.BaseConfig;
import com.willfp.eco.core.config.ConfigType;
import org.jetbrains.annotations.NotNull;

public class AttackSpeedConfig extends BaseConfig {
    public AttackSpeedConfig(@NotNull final PluginLike plugin) {
        super("attackspeed", plugin, true, ConfigType.YAML);
    }
}

File: eco-extensions/attackspeed/src/main/kotlin/com/willfp/ecoskills/attackspeed/AttackSpeedConfig.kt
package com.willfp.ecoskills.attackspeed

import com.willfp.eco.core.EcoPlugin
import com.willfp.eco.core.config.yaml.YamlExtendableConfig

class AttackSpeedConfig(plugin: EcoPlugin) : YamlExtendableConfig(
    "attackspeed",
    true,
    plugin,
    AttackSpeedConfig::class.java,
    "stats/"
) {
}